src/idfxt/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+set(SRC 
+  JSONDataInterface.cpp
+  JSONDataInterface.h
+  idd-full.h
```

Comment:
my current thoughts, and open questions, on this topic are already in this
thread.  We'll have to settle this with discussion.

On Fri, Jan 16, 2015 at 6:45 PM, Dan Macumber <notifications@github.com>
wrote:

> In src/idfxt/CMakeLists.txt
> <https://github.com/NREL/EnergyPlus/pull/4636#discussion-diff-23117783>:
>
> > @@ -0,0 +1,14 @@
> > +
> > +set(SRC
> > +  JSONDataInterface.cpp
> > +  JSONDataInterface.h
> > +  idd-full.h
>
> More importantly how will these be maintained? Is this where developers
> will add schema changes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NREL/EnergyPlus/pull/4636/files#r23117783>.
>